elisp-bytecode.texi

```diff
@@ -1878,62 +2185,79 @@ Make a binding recording buffer, point, and mark.
 @item @verb{|0161|}
 @tab @verb{| 113|}
 @tab @verb{|  set-buffer|}
-@tab Call @code{set-buffer} with one argument.
```

Comment:
Why was this removed? Is it incorrect?

Reply:
Probably an inadvertent mistake. Have readded and will push soon.

Reply:
Addressed in 8a3903d7 . Not sure why this still shows. Also, I think we should remove the "with one argument" stuff as those are indicated in the stack effects. In this table, I think we want just the minimum and the full description in the Instructions section. 

I would like a link with no extra verbiage which is why I started the other index. However I haven't been able to pull that off yet.

Reply:
I agree on th "with one argument" thing, it'd be best to omit both that and the descriptions and just write "call *" or "call +"